### ISSUE TITLE
Providers absolute files are not working

### ISSUE THREAD

**Issue by danielsantiago** [promptfoo/promptfoo#960]
Hello,

I am trying to add a custom provider with an absolute path but it is not working, if I include it with a relative path and run promptfoo in the specific path then it works without problems. I am following the example of: https://www.promptfoo.dev/docs/providers/custom-api#multiple-instances-of-the-same-provider

When I use:
```
providers:
  - id: file:///Users/myuser/project/provider.js
```

I got the following error when executing promptfoo from `/Users/myuser/project`

`Serialized Error: { code: 'ERR_MODULE_NOT_FOUND', url: 'file:///Users/myuser/project/Users/myuser/project/provider.js' }`

You can see that promptfoo is ignoring the three slashes from file (file:///) and is treating the path as relative rather than absolute.

**Comment by danielsantiago** [comment]
I think it has to do with this logic that is not considering that the path can already be absolute:

https://github.com/promptfoo/promptfoo/blob/13a970d2591a13e364b08ca976b2d4ec2ee3a3e1/src/util.ts#L294

**Comment by typpo** [comment]
Thanks for the report @danielsantiago!  Fix is in #966

**Comment by typpo** [comment]
Released in 0.65.2
